community/releases/release-manager-runbook/index.md

```diff
@@ -0,0 +1,329 @@
+   
+   This looks for a currently active release with this major and minor version.
+   It copies all parameters from that release (name, changelog gist, minimum
+   Halyard version), increments the patch version, and triggers
+   Publish_SpinnakerRelease with these parameters. In general, this is exactly
+   the behavior we want, but if you need to override this behavior (such as to
+   increment the minimum Halyard version in a patch release), you can call
+   Publish_SpinnakerRelease directly and pass the exact parameters that you’d
+   like the new release to have.
+
+4. After the job has completed, run ```hal version list``` and verify that the
+version you just released is listed, and the prior patch release for the minor
+version is no longer listed.
+
+5. Go to to [spinnaker.io](https://www.spinnaker.io/community/releases/versions/)
+and verify the following (leaving time for the site to rebuild):
+   
+   a. The version you just released is listed.
+   
+   b. The changelog for the new version looks correct.  It should start with the
+   changelog for the specific patch release, then list the changelog for each
+   patch release of the minor version in reverse order.
+   
+   c. The prior patch release for the minor version has been moved to the
+   “Deprecated Versions” section.
+    
+6. Approve the spinnaker-announce email (link will come in email).
+
+7. Ping the #spinnaker-releases channel to let them know that the new patch is
+available.
+
+## Release minor-version Halyard (repeat every 2-4 weeks as needed)
+
+1. Check for outstanding PRs.
+
+2. Run Flow_BuildAndValidate, selecting `nightly` Halyard. This will
+automatically check the “build Halyard” checkbox in the downstream 
+Build_PrimaryArtifacts flow.
+
+3. After that passes, navigate to:
+https://builds.spinnaker.io/job/Build_PrimaryArtifacts/${JOB_NUMBER}/artifact/build_output/build_halyard/last_version_commit.yml/*view*/
+(insert correct JOB_NUMBER) and copy the version (it will be the entire string prior to the colon).
+
+4. Run Publish_HalyardRelease:
+
+    a. Set `HALYARD_BUILD_VERSION_TO_RELEASE` to the version copied from the
+    prior step.
+
+## Release patch-version Halyard (repeat as needed)
+
+1. Run Build_Halyard:
+
+    a. Set **GITHUB_REPO_BRANCH** to the release branch of Halyard
+    (e.g., release-1.20.x).
+    
+    b. Set **OVERRIDE_PROCESS_GITHUB_REPO_BRANCH** to `master`.
+
+2. Run Publish_HalyardRelease:
+
+    a. Set **HALYARD_BUILD_VERSION_TO_RELEASE** to pre-colon output from
+    `last_version_commit.yml` of the prior job.
+    
+## Publish a new version of Spinbot (repeat as needed)
+
+1. Ensure no build is currently in progress.
+
+2. From your machine, run:
+   
+    ```
+   git clone https://github.com/spinnaker/spinnakerbot \
+   && cd spinnakerbot
+   ```
+   
+   Ensure you’re on the `master` branch:
+   
+   ```
+   git checkout master && git pull origin master
+   ```
+   
+   Publish Spinbot:
+   
+   ```
+    make docker
+   ```
```

Comment:
@ttomsu Doesn't Spinnakerbot get auto-deployed now that it's a cloud run job?

Reply:
Yes - spinbot runs right off the `master` branch now

Reply:
Oh that's right! I'll delete this section.